src/components/ui/field/FieldLabel.vue

```
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+import type { HTMLAttributes } from "vue"
+import { cn } from "@/lib/utils"
+import { Label } from '@/components/ui/label'
```

ESLint

[error] Expected "`@/components/ui/label`" to come before "`@/lib/utils`".

(perfectionist/sort-imports)